model/Models/Database.cs

```diff
@@ -59,13 +59,25 @@ public Database(string name, IList<string> filteredTypes = null)
 		public const string SqlWhitespaceOrCommentRegex = @"(?>(?:\s+|--.*?(?:\r|\n)|/\*.*?\*/))";
 		public const string SqlEnclosedIdentifierRegex = @"\[.+?\]";
 		public const string SqlQuotedIdentifierRegex = "\".+?\"";
-
 		public const string SqlRegularIdentifierRegex = @"(?!\d)[\w@$#]+";
 			// see rules for regular identifiers here https://msdn.microsoft.com/en-us/library/ms175874.aspx
 
-		#region " Properties "
+	    public static readonly string AutoGenerateComment = String.Format(
+	        @"//----------------------------------------------------------------------------
+// <auto-generated>
+//     This file was generated by schemazen.
+//     Date: {0}
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//----------------------------------------------------------------------------
+
+", DateTime.Today.ToString("MM-dd-yyyy"));
```

Comment:
Would it be appropriate to use the .ToShortDateString() method of DateTime to use the current cultures settings for date output?